ts/parts-3d/Series.ts

```diff
@@ -87,6 +111,7 @@ H.Series.prototype.translate3dPoints = function (): void {
             y: rawPoint.plotY as any,
             z: rawPoint.plotZ as any
         });
+        series.rawPoints = H.extend([], rawPoints);
```

Comment:
Any reason to use extend/merge here? 3D is quite heavy for the CPU already and this will just lower the performance.

Reply:
Yes, from line 123 we are changing value of `rawPoint` so I have created a copy to this change in series `rawPoints` array. However, for now I think that easier is to simply create new point object instead of changing `rawPoint` (so the naming will be correct).